projects/kubernetes/autoscaler/Makefile

```diff
@@ -16,6 +16,18 @@ FIX_LICENSES=$(REPO)/cluster-autoscaler/LICENSE
 HAS_RELEASE_BRANCHES=true
 EXCLUDE_FROM_STAGING_BUILDSPEC=true
 
+HAS_HELM_CHART=true
+
+HELM_GIT_TAG=$(shell cat ./$(RELEASE_BRANCH)/HELM_GIT_TAG)
+HELM_TAG=$(subst cluster-autoscaler-chart-,,$(HELM_GIT_TAG))
+HELM_DIRECTORY=$(RELEASE_BRANCH)
```

Comment:
```suggestion
```

Reply:
I need to override this for my release-branching to work. The chart will exist in the release branched directories, not the make root.

Can you help me understand why you don't want this overridden?

Reply:
Amended original commit adopting change 👍